src/sentry/workflow_engine/models/workflow.py

```diff
@@ -36,7 +38,8 @@ class Workflow(DefaultFieldsModel, OwnerModel, JSONConfigBase):
 
     @property
     def CONFIG_SCHEMA(self) -> dict[str, Any]:
```

Comment:
Nit: Not related to this PR, but we don't usually use all caps for class properties or methods. Usually saved for constants at the module level